### ISSUE TITLE
Entire Section Missing from Content SEO settings

### ISSUE THREAD

**Issue by CreateSean** [nystudio107/craft-seomatic#632]
Craft  3.4.18
SEOmatic 3.3.2

I have an entire section missing from the content SEO section of SEOmatic. The section exists, has urls, and has 138 entries. I do have category group with the same name as the missing channel. That category group is displaying.

There is no pagination on the content SEO page says it is displaying 1-18 of 18 items

**Comment by leyendeckermarc** [comment]
Any news on this? Same problem here!

**Comment by tobylinden** [comment]
We're encountering the same issue. All Content SEO data is missing along with the sitemap automatically created with SEOmatic. Any help would be appreciated!

Craft Pro 3.4.18
SEOmatic 3.3.2

**Comment by rgdekoning** [comment]
Craft Pro 3.4.18
SEOmatic 3.3.2

Also encountering same issue: Content SEO list is empty
occurring after update from SEOmatic 3.3.1 to 3.3.2

![Screenshot 2020-05-12 at 15 11 15](https://user-images.githubusercontent.com/790531/81695593-21ebdc00-9463-11ea-99e8-6bb6bcd335d6.png)

From storage/logs/web.log:
```
SQLSTATE[42000]: Syntax error or access violation: 1055 Expression #1 of SELECT list is not in GROUP BY clause and contains nonaggregated column 'DBNAME.seomatic_metabundles.id' which is not functionally dependent on columns in GROUP BY clause; this is incompatible with sql_mode=only_full_group_by
The SQL being executed was: SELECT *
FROM `seomatic_metabundles`
WHERE (`sourceBundleType` != '__GLOBAL_BUNDLE__') AND (`sourceSiteId`='1')
GROUP BY `sourceId`
ORDER BY `sourceName`
LIMIT 20
```

**Comment by khalwat** [comment]
Fixed in SEOmatic 3.3.3 -> https://github.com/nystudio107/craft-seomatic/releases/tag/3.3.3

**Comment by ScottHaggar** [comment]
I seem to be having this, or a similar issue with 3.3.4. One channel was lost when upgrading. 
<img width="705" alt="No_Blog_in_Content_SEO" src="https://user-images.githubusercontent.com/4719856/82929979-d39a0b00-9f41-11ea-9566-7d8f7cb1e6ed.png">
<img width="1127" alt="Channel setting" src="https://user-images.githubusercontent.com/4719856/82929985-d72d9200-9f41-11ea-8a5f-60f8baf333ca.png">

**Comment by khalwat** [comment]
And fixed again in 3.3.6 :/
